pkg/virt-handler/rest/lifecycle.go

```diff
@@ -87,6 +88,11 @@ func (lh *LifecycleHandler) UnpauseHandler(request *restful.Request, response *r
 	err = client.UnpauseVirtualMachine(vmi)
 	if err != nil {
 		log.Log.Object(vmi).Reason(err).Error("Failed to unpause VMI")
+		if strings.Contains(err.Error(), "LibvirtError(Code=55") {
+			response.WriteError(http.StatusConflict, fmt.Errorf("Cannot unpause VM during migration"))
```

Comment:
@SamAlber I don't think it will always be correct to conclude that this operation was failed because of a live migration. We can rely only on the error message that is returned from the endpoint(libvirt) and in this case we don't have any information whether the failure was because of live-migration or not.
And as was previously discussed, observing the state of the VM while executing this command will always be raceful.
Lets return the libvirt error message "domain is not runnig" but lets drop the `Code=55` and `Domain=22` messages, just send the value of `Message=` 
We can do a follow-up effort to enrich the returned error by translating the codes into messages and append them to the current error.